docs/installation_configuration_guide/ModelConfiguration.md

```diff
@@ -150,20 +150,22 @@ Important thing to remember is that Kafka server addresses/schema registry addre
+| topicsExistenceValidationConfig.validatorConfig.topicsFetchCacheTtl         | Low        | duration | 30 seconds       | TTL for caching list of existing topics                                                                                                                                                                                                                      |
+| topicsExistenceValidationConfig.validatorConfig.adminClientTimeout          | Low        | duration | 500 milliseconds | Timeout for communicating with Kafka cluster                                                                                                                                                                                                                 |
+| schemaRegistryCacheConfig.availableSchemasExpirationTime                    | Low        | duration | 10 seconds       | How often available schemas cache will be invalidated. This determines the maximum time you'll have to wait after adding new schema or new schema version until it will be available in Designer                                                             |
+| schemaRegistryCacheConfig.parsedSchemaAccessExpirationTime                  | Low        | duration | 2 hours          | How long parsed schema will be cached after first access to it                                                                                                                                                                                               |
+| schemaRegistryCacheConfig.maximumSize                                       | Low        | number   | 10000            | Maximum entries size for each caches: available schemas cache and parsed schema cache                                                                                                                                                                        |
+| lowLevelComponentsEnabled                                                   | Medium     | boolean  | false            | Add low level kafka components: 'kafka-json', 'kafka-avro', 'kafka-registry-typed-json'                                                                                                                                                                      |
+| avroAsJsonSerialization                                                     | Low        | boolean  | false            | Send payload on avro schema topic as plaintext json                                                                                                                                                                                                          |
```

Comment:
`Send and recieve json messages conformed to Avro schema`
or
`Send and recieve json messages serialized using Avro schema` ?